lib/simctl.js

```diff
@@ -32,7 +27,14 @@ async function launch (udid:string, bundleId:string):void {
 
 async function createDevice (name:string, deviceTypeId:string,
     runtimeId:string):void {
-  await simExec("create", 0, [name, deviceTypeId, runtimeId]);
+
+  let out;
+  try {
```

Comment:
"create" is the only one which returns besides "getDevices" which is heavily parsed already.

The try/catch just gives a "nice" error. Do you think we should add it to all of them? They will still throw an error just maybe a little less verbose.
Hm, I guess it is good to do it to all of them then, I'll just modify simExec.